lms/static/js/views/fields.js

```diff
@@ -74,14 +104,14 @@
             },
 
             title: function (text) {
-                return this.$('.u-field-title').html(text);
+                return HtmlUtils.setHtml(this.$('.u-field-title'), text);
```

Comment:
When we want escaping and we are not working with HTML, it is simpler to just change `.html(text)` to `.text(text)`.  I suspect this is the case.  If it isn't, then we should rename that variable `text` to `html`.